receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -267,12 +271,13 @@ func (jr *jReceiver) PostSpans(ctx context.Context, r *api_v2.PostSpansRequest)
 	}
 
 	ctx = obsreport.ReceiverContext(ctx, jr.id, grpcTransport)
-	ctx = obsreport.StartTraceDataReceiveOp(ctx, jr.id, grpcTransport)
+	jr.obsrecv = obsreport.NewReceiver(obsreport.ReceiverSettings{ReceiverID: jr.id, Transport: grpcTransport})
```

Comment:
Same here. Or have different instances per transport is better.

Reply:
Initialized in ctor with different instances per transport